x-pack/plugins/security_solution/public/timelines/components/timeline/data_providers/index.tsx

```diff
@@ -93,6 +93,15 @@ export const DataProviders = React.memo<Props>(({ timelineId }) => {
     (state) => (getTimeline(state, timelineId) ?? timelineDefaults).dataProviders
   );
   const droppableId = useMemo(() => getDroppableId(timelineId), [timelineId]);
+  useEffect(() => {
```

Comment:
I wonder if this should be a custom hook. E.g. `useFocusReset()` or sth.

Reply:
that's a good idea, i will do that